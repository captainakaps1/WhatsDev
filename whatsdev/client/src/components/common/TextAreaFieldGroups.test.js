import TextAreaFieldGroup from "./TextAreaFieldGroup";
import { render } from "@testing-library/react";

describe("Component Test - InputGroup", () => {
  const data = {
    name: "",
    placeholder: "",
    value: "",
    error: "",
    onChange: () => {},
    info: "",
  };

  it("matches snapshot", () => {
    const { asFragment } = render(<TextAreaFieldGroup {...data} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders without crashing", async () => {
    const { findByTestId } = render(<TextAreaFieldGroup {...data} />);
    expect(await findByTestId("Test-Area-Field-Group-div")).toBeDefined();
  });
});
