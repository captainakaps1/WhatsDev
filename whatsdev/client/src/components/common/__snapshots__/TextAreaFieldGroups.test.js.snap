// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Test - InputGroup matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="form-group"
    data-testid="Test-Area-Field-Group-div"
  >
    <textarea
      class="form-control form-control-lg"
      name=""
      placeholder=""
    />
  </div>
</DocumentFragment>
`;
